apps/cli/tf/bucket.tf

```diff
@@ -0,0 +1,108 @@
+resource "aws_s3_bucket" "sunodo_cli_bucket" {
+  bucket        = var.bucket_name
+  force_destroy = false
+
+  tags = {
+    Name        = "sunodo_cli_bucket"
+    Environment = "Production"
+  }
+}
+
+resource "aws_s3_bucket_acl" "sunodo_cli_bucket_acl" {
+  bucket = aws_s3_bucket.sunodo_cli_bucket.id
+  acl    = "public-read"
```

Comment:
I don't think the bucket itself must be public read.
It's better to have some domain (`cli.sunodo.io`) in a CloudFront web distribution, and S3 behind it.

Reply:
Agree, working on it ...

Reply:
Fixed !